DeusEx/Classes/DXRFixup.uc

```diff
@@ -39,15 +39,22 @@ function BuffScopes()
     }
 }
 
-function FixUnbreakableCrates()
+function FixUnbreakableObjects()
```

Comment:
maybe we could make this config-driven too, a list of class<DeusExDecoration> along with a number of hitpoints to set, and maybe also minDamageThreshold... other fun variables we could override
```
var() class<Fragment> fragType;
var() bool bFlammable;				// can this object catch on fire?
var() float Flammability;			// how long does the object burn?
var() bool bExplosive;				// does this object explode when destroyed?
var() int explosionDamage;			// how much damage does the explosion cause?
var() float explosionRadius;		// how big is the explosion?
```
also bPushable, which I think is part of the Decoration base class